walkthrough/garden_cookiecutter/{{cookiecutter.directory_name}}/{{cookiecutter.short_name}}.py

```diff
@@ -1,28 +1,20 @@
 import json
-from pathlib import Path
-from typing import Dict, List
+from typing import List
 
 import pandas as pd
 from owid.catalog import Dataset, Table
 from owid.catalog.utils import underscore_table
+from owid.datautils import geo
 from structlog import get_logger
 
+from etl.helpers import Names
 from etl.paths import DATA_DIR
 
 log = get_logger()
 
+# naming conventions
+N = Names(__file__)
```

Comment:
`__file__` does not work when working on a jupyter notebook, which is annoying. What I usually do to work around this is to create a `shared.py` file that contains the `__file__` (and possibly other definitions, like log). I don't know if you like this idea or if there is a better work around. For me it's like having an `__init__.py` on every module.

Reply:
Eh, that's right. `shared.py` is a really good solution, but I don't want to complicate walkthrough even more (also given that you're the only one using notebooks this way).

Alternative is to use ugly
```python
from pathlib import Path
N = Names(globals().get('__file__', (Path() / "dummy.py").resolve()))
```
which falls back to working directory + hardcoded module name if `__file__` doesn't exist.